scrape/scrape.go

```diff
@@ -445,13 +472,40 @@ func (sp *scrapePool) sync(targets []*Target) {
 		}
 	}
 
+	targetScrapePoolTargetsAdded.WithLabelValues(sp.config.JobName).Set(float64(len(uniqueLoops)))
```

Comment:
How are these removed when a scrape config is removed?

A custom collector is probably a better way to do this.

Reply:
We do have two other metrics per scrape config. I now unregister them on stop.